torchaudio/functional/filtering.py

```diff
@@ -981,11 +1001,16 @@ def overdrive(waveform: Tensor, gain: float = 20, colour: float = 20) -> Tensor:
 
     output_waveform = torch.zeros_like(waveform, dtype=dtype, device=device)
 
-    # TODO: Implement a torch CPP extension
-    for i in range(waveform.shape[-1]):
-        last_out = temp[:, i] - last_in + 0.995 * last_out
-        last_in = temp[:, i]
-        output_waveform[:, i] = waveform[:, i] * 0.5 + last_out * 0.75
+    # # TODO: Implement a torch CPP extension
+    # for i in range(waveform.shape[-1]):
+    #     last_out = temp[:, i] - last_in + 0.995 * last_out
+    #     last_in = temp[:, i]
+    #     output_waveform[:, i] = waveform[:, i] * 0.5 + last_out * 0.75
```

Comment:
I think you can get rid of these comments now.

Reply:
@mthrok sure will remove them.
Also I will try to implement the ```parallel_for``` as mentioned by @cpuhrsch in the old PR comments ( #580 ).

Reply:
@bhargavkathivarapu sounds good. Could you try measure the performance improvement?
Something like the following and change the shape of tensor to variety of shapes.
If `numactl` is missing, you can install it with `sudo apt-get install -y numactl` on Ubuntu.

```
OMP_NUM_THREADS=1 numactl --membind 0 --cpubind 0 python -m timeit -n 100 -r 5 -s """
import torch;
import torchaudio
x = torch.zeros(32, 100, dtype=torch.float)
""" """
torchaudio.functional.overdrive(x)
"""
```